### DISCUSSION TITLE
Remove warmup seconds for warps that don't exist

### DISCUSSION THREAD

**Discussion by Redwolf223** [Zrips/CMI#7975] [Enhancement]
It would be great to have the same warmup for the /warp command as the one for /tpa.

Currently if you use /warp with a non existing warp name it will start a warmup which is not ideal as the player has to wait before getting the message "that warp doesn't exist".

We are using the warp system with warmup.

**Comment by LiquidEagle** [comment]
Bumping this, i would consider this as a bug not an enhancement.
